staker/block_validator.go

```diff
@@ -105,7 +108,26 @@ func (c *BlockValidatorConfig) Validate() error {
 		}
 		c.memoryFreeLimit = limit
 	}
-	return c.ValidationServer.Validate()
+	if c.ValidationServerConfigs == nil {
+		if c.ValidationServerConfigsList == "default" {
+			c.ValidationServerConfigs = []rpcclient.ClientConfig{c.ValidationServer}
+		} else {
+			var validationServersConfigs []rpcclient.ClientConfig
+			if err := json.Unmarshal([]byte(c.ValidationServerConfigsList), &validationServersConfigs); err != nil {
```

Comment:
I'm concerned about this part. Does this mean that we will have to have an escaped json string as a string in the config? Doing this was a mistake by me in the DAS configuration and is horrible UX for people configuring nitro.

Reply:
Nope, the `ValidationServerConfigsList` field is only ever to be used as a command line argument and not inside config. We make sure this is the case by checking if `c.ValidationServerConfigs == nil` and only then proceed to access `ValidationServerConfigsList` field. 
(in odd cases when user supplies both fields in the config file, the `ValidationServerConfigs` is chosen)

And as for configuring `ValidationServerConfigs` field in the config heres an example: 
`"node":{"block-validator":{"validation-server-configs":[{"url":"abc.com","timeout":123456}]}}`

Reply:
The reason for doing this was there was no straightforward way to supply the server configs via command line as an array, and having a option that takes a json string is very close how one would supply this field (ValidationServerConfigs) in the config file. 

Here's an example for command line usage:
`./target/bin/nitro --parent-chain.connection.url <url> --chain.id=421614 --node.block-validator.enable --validation.wasm.enable-wasmroots-check=false --node.block-validator.validation-server-configs-list='[{"url":"abc.com","timeout":123,"arg-log-limit":10},{"retry-delay":156}]'`

Reply:
Okay, this is actually great, and probably the same pattern we should use when we need arrays of config to work on the command line. Nice work figuring this out.